### ISSUE TITLE
problem getting libimobiledevice 1.3

### ISSUE THREAD

**Issue by tomsarra** [libimobiledevice/libimobiledevice#1206]
I am using Ubuntu 20.04 and need libimobiledevice for another app , it wants libimobiledevice 1.3 but no matter where i get it , cloning from githib , or getting it at libimobiledevice.org, I get libimobiledevice 1.2. I am not sure what I am doing wrong.
thank you for any help you can provide
Tom S

**Comment by tomsarra** [comment]
Thank you for the quick response, I am not sure if I am to answer direct email or not.
I am now using Linux, synaptic shows libimobile packages but none are installed. 
I followed the instructions on the GitHub libimobile page , the dependencies first , then cloned libimobile , cd , make and installed, all went well. However I checked in synaptic and now shows libilemobiledevice6 installed showing version 1.2.
Maybe I should use synaptic to completely remove libimobile .
Ok back to it
Tom S

**Comment by mexmer** [comment]
yes, remove libimobiledevice from synaptic. synaptic is package manager and downloads package from distribution repository. 

packages from distribution repository will have priority over manual compiled packages.

**Comment by tomsarra** [comment]
Yes I will do that
Thanks
Tom S
